### ISSUE TITLE
Design page keeps loading

### ISSUE THREAD

**Issue by yarin-zhang** [Budibase/budibase#7749]
Budibase version: 1.3.15 (Docker deployment)
Chrome version: 105.0.5195.102
Safari version: 15.6.1 (17613.3.9.1.16)

Issue Description

I can't get to the Budibase Design page, it always shows up as loading. And it doesn't show up in the Chrome Console either. I also can't get to the Preview page.

![image](https://user-images.githubusercontent.com/58888890/189847979-a27e8d3f-4171-42d4-a6db-37ebddcf5060.png)

![image](https://user-images.githubusercontent.com/58888890/189848667-edfa8b06-0c23-44a9-9dd0-75cfc096b972.png)

![image](https://user-images.githubusercontent.com/58888890/189848834-be0a4efa-2647-4006-8221-22d9e4260526.png)

_Originally posted by @yarin-zhang in https://github.com/Budibase/budibase/issues/7723#issuecomment-1245061003_

**Comment by aderici** [comment]
you can see the "preview" request on "network" tab. Can you check and see what it returns? Any error messages in "console" tab?

**Comment by yarin-zhang** [comment]
I don't see any valuable hints, including in the console and network tab.

The only hint that appears is the Zotero Chrome plugin, which still doesn't show anything after I disabled it, which I think is an unrelated factor.

![image](https://user-images.githubusercontent.com/58888890/190378961-7aac07cd-bd84-4a67-9b20-25ace05074d0.png)
![image](https://user-images.githubusercontent.com/58888890/190378272-bddb4a56-5195-496d-9f90-0d50dbc291ac.png)
![image](https://user-images.githubusercontent.com/58888890/190378708-7239ac8c-e009-401c-9da0-3b662430bd6f.png)
![image](https://user-images.githubusercontent.com/58888890/190379083-a7ba130b-2525-464d-a0a0-2b62e3ed7917.png)

**Comment by aaronosher** [comment]
I think I'm having the same version. This is a brand new application and this is what the preview endpoint is showing. Nothing useful in the console. Nothing particularly helpful in the server logs either, this is on v1.3.20.

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/4386583/190714807-dbb14085-f230-4d29-bedd-0144e284d324.png">

**Comment by charlfields** [comment]
Same problem with me:
Budibase version: 1.4.3 (Docker deployment)
Chrome version: 105.0.5195.127 
Safari version: 15.4 (17613.1.17.1.6)

**Comment by yarin-zhang** [comment]
This problem still exists in Budibase version 2.4.4 (Docker deployment).

I found a very strange phenomenon, when I am in Firefox Developer Edtion, the page is displayed normally.

![image](https://user-images.githubusercontent.com/58888890/224204267-6071c7e8-53ed-42e5-91cb-f1f33c987c1d.png)

But when I'm in Chrome it keeps loading. I tried guest mode (without any browser plugins) and it also keeps loading.

![image](https://user-images.githubusercontent.com/58888890/224204464-71fa306c-b5b7-463b-8189-2ebeed7b4a46.png)
